include/oneapi/dpl/internal/dynamic_selection_impl/sycl_backend.h

```diff
@@ -0,0 +1,143 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ONEDPL_SYCL_BACKEND_IMPL_H
+#define _ONEDPL_SYCL_BACKEND_IMPL_H
+
+#include <CL/sycl.hpp>
+#include "oneapi/dpl/internal/dynamic_selection.h"
```

Comment:
Seems like a cyclic dependency, because `dynamic_selection.h` in turn includes this file.

Reply:
All includes are being reviewed to remove unnecessary and cyclic includes.